src/common/snippets/tests/src/pass/collapse_subgraph.cpp

```diff
@@ -26,14 +26,22 @@ void CollapseSubgraphTests::run() {
             });
 }
 
-TEST_F(CollapseSubgraphTests, smoke_Snippets_Eltwise) {
+class SKIP_CollapseSubgraphTests : public CollapseSubgraphTests {
+public:
+    void SetUp() override {
+        GTEST_SKIP();
+    }
+    void TearDown() override{};
+};
+
+TEST_F(SKIP_CollapseSubgraphTests /* CVS-114607 */, smoke_Snippets_Eltwise) {
```

Comment:
If I remember correctly, we have some config files to disable tests
I believe it's called "skip_tests_config.cpp" or something like that 
could try this approach?

Reply:
It doesn't work with `TransformationTestsF::TearDown()` which throws on missing model.